roles/edpm_deploy/tasks/main.yml

```diff
@@ -18,7 +18,6 @@
   when:
     - zuul is not defined
     - cifmw_operator_build_output is defined
-    - "'dataplane-operator' in operators_build_output.operators"
```

Comment:
I think we can drop this whole task "Define minimal set of repo variables when not running on Zuul" as it is specific to dataplane operator.